src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +47,10 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+
+        return numbers.stream()
+                .filter(number -> number > 3)
+                .map(number -> number * 2)
+                .reduce(0, (x, y) -> x + y);
```

Comment:
map 대신 mapToInt 를 활용하면 reduce 대신 sum() 최종 연산도 가능합니다 😃 

```suggestion
                .mapToInt(number -> number * 2)
                .sum();
```